test/python/providers/test_fake_backends.py

```diff
@@ -416,6 +417,9 @@ def __init__(self, num_ctrl_qubits, ctrl_state=None):
         from qiskit_aer.noise.noise_model import QuantumErrorLocation
 
         sim = AerSimulator()
+        # test only if simulator's backend is V1
+        if _get_backend_interface_version(sim) > 1:
```

Comment:
You can just do:

```suggestion
        if sim.version > 1::
```

The version attribute is the only shared attribute with the `Backend` abstract interface that both `BackendV1` and `BackendV2` inherit from. The reason `_get_backend_interface_version()` existed originally was because the predecessor to `BackendV1`, `BaseBackend`, was not a versioned interface and didn't have a `version` attribute. (it did have a method `version()` if I remember correctly that would return other metadata, so it couldn't be used or overridden for version negotiation).